string need_name            # 需求类型
string person_name          # 用户姓名
string IDtype               # 用户身份
float64 target_angle        # 交互对象角度
float64 target_distance     # 交互对象距离
string rob_emotion          # 机器人情绪类型
int8 rob_emotion_intensity  # 机器人情绪强度
string person_emotion       # 用户的心情
float64 weight              # 需求权重
string speech               # 回答文本[机器人回复的话]
int64 qt_order              # qt显示界面中的索引
int8 satisfy_value          # 需求自我满足值[用于情感模块中的自我评价]